### ISSUE TITLE
Instructions to add your user to the ssl-cert group

### ISSUE THREAD

**Issue by marclava** [kasmtech/KasmVNC#241]
Not working::
sudo addgroup $USER ssl-cert
Working:
sudo usermod -a -G ssl-cert $USER

**Comment by mmcclaskey** [comment]
The current instructions are the following for Ubuntu/Debian, which works at least on ubuntu, I have not tested all flavors.
```bash
sudo adduser $USER ssl-cert
```

**Comment by GogoFC** [comment]
> The current instructions are the following for Ubuntu/Debian, which works at least on ubuntu, I have not tested all flavors.
> 
> ```shell
> sudo adduser $USER ssl-cert
> ```

Yeah the [Docs](https://www.kasmweb.com/kasmvnc/docs/1.0.0/install.html#debian-ubuntu-kali) have it wrong saying `sudo addgroup $USER ssl-cert`
